ui/component/postViewer/view.jsx

```diff
@@ -104,8 +104,10 @@ function PostViewer(props: Props) {
 
       <ClaimAuthor uri={uri} />
 
-      <FileRenderInitiator uri={uri} />
-      <FileRenderInline uri={uri} />
+      <div className="file-render--post-container">
+        {<FileRenderInitiator uri={uri} />}
```

Comment:
whatis this enclosure doing?

Reply:
Enforcing a minimum height.  The 2 childs have sub-childs that come and go, so added an enclosure instead of tweaking every child's css